modules/cli/src/main/scala-2.12/coursier/cli/params/ResolveParams.scala

```diff
@@ -18,7 +19,7 @@ final case class ResolveParams(
   benchmarkCache: Boolean,
   tree: Boolean,
   reverseTree: Boolean,
-  whatDependsOn: Set[Module],
+  whatDependsOn: Set[ModuleMatcher],
```

Comment:
Nit: I guess we can make that a simple `Seq` now (it was a `Set` only to check for inclusion, which we don't do anymore now).